frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -776,5 +786,43 @@ export const eventUsageLogic = kea<
         reportRecordingPlayerSkipInactivityToggled: ({ skipInactivity }) => {
             posthog.capture('recording player skip inactivity toggled', { skip_inactivity: skipInactivity })
         },
+        reportNewExperimentButtonClicked: () => {
+            posthog.capture('new experiment button clicked')
+        },
+        reportExperimentCreated: ({ experiment }) => {
+            posthog.capture('experiment created', {
+                name: experiment.name,
+                id: experiment.id,
+                filters: experiment.filters,
+                parameters: experiment.parameters,
+            })
+        },
+        reportExperimentViewed: ({ experiment }) => {
+            posthog.capture('experiment viewed', {
+                name: experiment.name,
+                id: experiment.id,
+                filters: experiment.filters,
+                parameters: experiment.parameters,
+            })
+        },
+        reportExperimentLaunched: ({ experiment, launchDate }) => {
+            posthog.capture('experiment launched', {
+                name: experiment.name,
+                id: experiment.id,
+                filters: experiment.filters,
+                parameters: experiment.parameters,
+                launch_date: launchDate.toISOString(),
+            })
+        },
+        reportExperimentCompleted: ({ experiment, endDate, duration }) => {
+            posthog.capture('experiment completed', {
+                name: experiment.name,
+                id: experiment.id,
+                filters: experiment.filters,
+                parameters: experiment.parameters,
+                end_date: endDate.toISOString(),
+                duration,
```

Comment:
I would include information on whether the experiment was significant when terminated.

Reply:
this is good, definitely